recipe/meta.yaml

```diff
@@ -203,8 +205,10 @@ outputs:
         - certifi
         # Test with the two currently supported lapack implementatons
         # One test done on different versions of python on each platform
-        - liblapack * *openblas         # [py==36]
+        - liblapack * *mkl              # [py==36]
         - liblapack * *mkl              # [py==37]
+        - libtiff 4.0                   # [py==36]
+        - libtiff 4.1                   # [py==37]
```

Comment:
Do we know why? Would it not be possible to align both to 4.1?

Reply:
I'm just trying to make it fail and to reproduce on a CI.

In the issue, I show the full environment that fails for me.

Reply:
Thanks for clarifying.

Reply:
I really don't understand what is the difference between my environments. Python 3.7 and libtiff 4.1 and opencv built a while back doesn't work :/

Reply:
I'm really not sure what is different. Maybe opencv is a red herring, and something else is depending on libtiff 4.0